docs/index.md

```diff
@@ -532,10 +532,10 @@ prefetch_related = {"serializer_field_name": "field_to_prefetch"}
 
 **Syntax Interpretation**
 
-* `serializer_field_name` stands for the name of the field to prefetch or select(as named on a serializer)
-* `fields_to_select` stands for argument(s) to pass when calling `select_related` method
-* `fields_to_prefetch` stands for arguments(s) to pass when calling `prefetch_related` method
-* If you want to select or prefetch nested field use dot(.) to separate parent and child fields on `serializer_field_name` eg `parent.child`
+* `serializer_field_name` stands for the name of the field to prefetch or select(as named on a serializer).
+* `fields_to_select` stands for argument(s) to pass when calling `select_related` method. This can be a string or `Prefetch` object.
+* `fields_to_prefetch` stands for arguments(s) to pass when calling `prefetch_related` method.
```

Comment:
Shouldn't 
>  This can be a string or `Prefetch` object.

be on `fields_to_prefetch` instead of `field_to_select`?.

Reply:
Yeah, sorry! Got it mixed around here.